modules/cnv_connecting_resource_bridge_network.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_attaching_resource_multiple_networks.adoc
+
+[id="connecting-resource-bridge-network-{context}"]
+= Connecting a resource to a bridge network
+
+As a network administrator, you can configure a NetworkAttachmentDefinition
+of type `cnv-bridge` to provide Layer-2 networking to Pods and virtual machines.
+
+.Prerequisites
+
+* {ProductName} 2.0 or newer
+* A Linux bridge must be configured on every host.
+* `vlan_filtering` must be enabled on the bridge.
```

Comment:
I'd use "should" instead of "must".

The world is not going to end if they forget it; they would simply be unable to use vlans.

Reply:
"Should" is a word we tend to avoid in docs because it is ambiguous in meaning. I'm going to say "vlan_filtering must be enabled on the bridge if you want to use VLANs." :)

Reply:
I'd rather you keep the simple shorter "must".

Reply:
Well, you brought up a good point that it isn't really a "must." So now I'm not sure. :) 

I agree that "vlan_filtering must be enabled on the bridge if you want to use VLANs" is not the most graceful way to phrase it... and it's in the passive voice (though that seems natural  to me in a prerequisites section)

More options:
"To use VLANs, enable vlan_filtering on the bridge."
"If you want to use VLANs, enable `vlan_filtering` on the bridge."
"`vlan_filtering` must be enabled on the bridge."
(and endless iterations thereof.)

paging @kalexand-rh - thoughts?

Reply:
Where do you use VLANs? I don't see where you specify them in the code block. Since it's a prereq, the bridge should already be enabled before you start this topic. Maybe "If you use VLANs, `vlan_filtering` must be enabled on the bridge."

+1 to not using should. From our style guide:
> :exclamation:  should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as
appropriate.